.gitignore

```diff
@@ -1,3 +1,2 @@
 node_modules
 .tempChromeProfileForDebug
-*.map 
```

Comment:
ohhhh my bad, you should revert this one, did not notice yesterday it was already in... so sorry